includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -18,6 +20,28 @@
  * Note: this task is separate to the Payments → Overview disputes task, which is defined in client/overview/task-list/tasks.js.
  */
 class WC_Payments_Task_Disputes extends Task {
+	/**
+	 * Client for making requests to the WooCommerce Payments API
+	 *
+	 * @var WC_Payments_API_Client
+	 */
+	private $api_client;
+
+	/**
+	 * Database_Cache instance.
+	 *
+	 * @var Database_Cache
+	 */
+	private $database_cache;
+
+	/**
+	 * WC_Payments_Task_Disputes constructor.
+	 */
+	public function __construct() {
+		$this->api_client     = \WC_Payments::create_api_client();
```

Comment:
This shouldn't be recreated, we can access the already constructed class via
```suggestion
		$this->api_client     = \WC_Payments::get_payments_api_client();
```